docs/api.rst

```diff
@@ -49,15 +49,6 @@ Networks
   :members:
   :undoc-members:
 
-Utilities
```

Comment:
Perhaps we could replace this with a "types" section to actually document them? (See my other comments.) Ditto the `HostConfig` object, and so on.